packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -19,8 +19,48 @@ class SharingCollection extends DocumentCollection {
       data: resp.data.map(normalizeSharing)
     }
   }
+  /**
+   *
+   * Creates a new Sharing. See https://docs.cozy.io/en/cozy-stack/sharing/#post-sharings
+   *
+   * @param {object} params
+   * @param {object} params.document The document to share
+   * @param {string} params.description Description of the sharin
+   * @param {string} params.previewPath The path (route) to see the preview
+   * @param {Array} params.rules The rules defined to the sharing. See https://docs.cozy.io/en/cozy-stack/sharing-design/#description-of-a-sharing
```

Comment:
I think it'd be helpful to have a `@typedef` here. This way we do not have to go look at a different file, and when we'll go to typescript it will be easier to migrate automatically the jsdoc to typescript definitions.


```jsx
@typedef Rule - A sharing rule
@property ...

* @param {Array<Rule>} params.rules ....
```

Reply:
30f5bb8bd5496a226c3938fdc03fc74976fc7a79